### DISCUSSION TITLE
How can I use SO iso image to update a SO standalone Redhat network install?

### DISCUSSION THREAD

**Discussion by TotieBash** [Security-Onion-Solutions/securityonion#12436] [2.4] [answered]
### Version

2.4.40

### Installation Method

Network installation on Red Hat derivative like Oracle, Rocky, Alma, etc.

### Description

upgrading

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

28

### Storage for /

150

### Storage for /nsm

250

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Asking for unsupported question:  How can I use SO iso image to update a SO standalone Redhat network install?

I built the system on RHEL 9.2 using standalone network install.  But after the initial install it is airgap from there forward.  I know I need to edit the soup script but I'm just asking if someone has successfully done the work here in this forum before I dive in and spending a day or two on the soup script.  My main goal is the docker container update and configurations, minus OS update patches.  The OS update and patches I will take care of it separately.  Any ideas would be appreciated.  Thanks

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
```
For most use cases, we HIGHLY recommend using our official Security Onion images as shown in the Installation section. Our official images are the only fully supported installation method and you should use them if any of the following apply to you:

- You are deploying in an enterprise environment.
**- You are deploying in an airgap environment.**
- You are performing a distributed deployment.
- You want the quickest and easiest installation with the fewest issues.
- You need full support.

```
https://docs.securityonion.net/en/2.4/network-installation.html

**Reply by TotieBash** [reply, marked as answer]
copy all thanks Chris...  I fully understand I was asking for unsupported question...  
But never mind I got it to work yesterday.
Summary fix:
1.  Fix soc_global.sls airgap=true
2.  `yum-config-manager --disable \*`
3. create local repo securityonion.repo  for /nsm/repo
4. copy OL9 /etc/os-release and paste it on RHEL9
5. `soup -f /dev/sr0`     
6. `cp /sbin/soup /root/soup`
7. `vi /root/soup`
- add "--nogpgcheck" to dnf and yum line entries
- search and remove "upgrade_salt()" section
- search and remove "Upgrading Salt" section
8. `/root/soup -f /dev/sr0` 
press "C" to skip os patches.
Quick checks appears to be all working.
